ray-operator/controllers/ray/rayjob_controller_unit_test.go

```diff
@@ -170,6 +170,19 @@ func TestGetSubmitterTemplate(t *testing.T) {
 		}
 	}
 	assert.True(t, found)
+
+	// Test 5: Check default FQ_RAY_IP env var
+	submitterTemplate, err = r.getSubmitterTemplate(rayJobInstanceWithTemplate, nil)
+	assert.NoError(t, err)
+	found = false
+	for _, envVar := range submitterTemplate.Spec.Containers[utils.RayContainerIndex].Env {
```

Comment:
Maybe we can make the function [envVarExists](https://github.com/ray-project/kuberay/blob/c45d959a2e149b640f292b1304dcbbb8289486f8/ray-operator/controllers/ray/common/pod.go#L656) public and reuse it? In the future, we can consider moving the function to `ray-operator/utils` instead.

Reply:
Can I make this change in a follow-up PR? There are other tests and assertions that I would have to refactor

Reply:
Sure. Would you mind opening an issue to track the progress? Thanks!